OOInteraction/src/commands/CSceneHandlerItemTest.cpp

```diff
@@ -149,6 +209,91 @@ Interaction::CommandResult* CSceneHandlerItemTest::execute(Visualization::Item*,
 		Visualization::VisualizationManager::instance().mainScene()->addTopLevelNode(newNode, 1);
 	}
 
+	for (auto id : movedNodes)
+	{
+		auto oldNode = const_cast<Model::Node*>(oldVersionManager->nodeIdMap().node(id));
+		auto newNode = const_cast<Model::Node*>(newVersionManager->nodeIdMap().node(id));
+
+		Visualization::VisualizationManager::instance().mainScene()->currentViewItem()
+				->addArrow(oldNode, newNode, QString{"move_arrows"});
+
+		Visualization::VisualizationManager::instance().mainScene()->addPostEventAction(
+							  [oldNode, newNode](){
+			auto items = Visualization::VisualizationManager::instance().mainScene()->currentViewItem()
+							->findAllVisualizationsOf(oldNode);
+			items.append(Visualization::VisualizationManager::instance().mainScene()->currentViewItem()
+							 ->findAllVisualizationsOf(newNode));
+			for (auto item : items)
+			{
+				auto overlay = new Visualization::HighlightOverlay{item};
+				QColor blue{"blue"};
```

Comment:
Ideally, instead of manually providing a color, use a predefined style.

Only fix if you plan on using the test command more.